Document Custom Policy Upgrade for Mule 3

fermujica opened the pull request.

fermujica committed:
Add steps to configure Mule 3 custom policies upgrade

fermujica committed:
Add links to upgrade custom policy

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
+
+== Uploading a New Version of Your Custom Policy
```
Should we point them to https://docs.mulesoft.com/api-manager/2.x/policy-mule3-creating-custom-policy#adding-the-custom-policy-to-api-manager ?

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+. Expand the policy name you want to upgrade, click *Actions*, and select *Change Version*.
++
+image::upgrade-custom-policy-mule3-3e0bb.png[align=center]
+. Select the new version of your custom policy, and click *Configure Policy*.
```
We should explain that in the list of the modal that opens, will list all custom policies.

So expect to see 
- Regex Filter Policy
- Regex Filter Policy v2
- Other Policy
- Other Policy-2
- Other Policy-new

sbehrends replied:
A popup will show an all the custom policies of tour business group will be listed (check this [Sergio])

sbehrends replied:
Same rules applies if a policy in the list has unmet requirements.

sbehrends commented on modules/ROOT/nav.adoc:
```diff
@@ -125,6 +125,7 @@
    **** xref:policy-mule3-custom-policy-references.adoc[Configuration and Definition File Reference]
    **** xref:policy-mule3-pointcut-reference.adoc[Pointcut Reference]
    **** xref:policy-mule3-resource-level-custom-policy.adoc[Enable a Resource Level Support for a Custom Policy]
+   **** xref:upgrade-custom-policy-mule3.adoc[Upgrade Custom Policies]
```
Change Custom Policy Version

sbehrends commented on modules/ROOT/pages/policy-mule3-creating-custom-policy.adoc:
```diff
@@ -199,5 +199,6 @@ After you have created the YAML and XML files, or downloaded the files, you make
 
 == See Also
 
+* xref:upgrade-custom-policy-mule3.adoc[Upgrade Custom Policies]
```
Change Custom Policy Version

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+= Upgrade Custom Policies for Mule 3
```
Change Custom Policy Version for Mule 3

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+include::_attributes.adoc[]
+endif::[]
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
```
Let's add what problem are we solving with this
https://docs.google.com/document/d/1RQZr_kLzyGyqItMzjyrBT1-0_H5plaNf5M-_QCoyzvQ/edit#heading=h.6ek3exazel76

In Api Manager you can change the custom policy version without leaving your API insecure.

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+image::upgrade-custom-policy-mule3-08395.png[align=center]
+. Click *Add*.
+
+== Upgrading your Custom Policy Version in your API Instance
```
Changing the version of your Custom Policy for an API Instance

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+
+== Upgrading your Custom Policy Version in your API Instance
+
+. In API Manager, go to *API Administration*, and select the version of the API instance you want to upgrade its custom policy.
```
change version of custom policy

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
+
+. In API Manager, go to *API Administration*, and select the version of the API instance you want to upgrade its custom policy.
+. Click *Policies*.
+. Expand the policy name you want to upgrade, click *Actions*, and select *Change Version*.
```
you want to change version

sbehrends commented on modules/ROOT/pages/upgrade-custom-policy-mule3.adoc:
```diff
@@ -0,0 +1,35 @@
++
+image::upgrade-custom-policy-mule3-0a3c0.png[align=center]
++
+Note that the fields that you configured on your earlier versions are still configured in your upgraded policy. Additionally, the UI points out the new fields to configure.
```
Change version of the policy.
UI will show a NEW tag besides new filed.
Current policy Sensitive information will be masked out.

fermujica committed:
'upgrade' should be 'change versions'

fermujica committed:
Update instructions to upload custom policy

This also removes duplicated steps in custom policy change versions 
instructions

fermujica committed:
Add missing link

sbehrends commented on modules/ROOT/nav.adoc:
```diff
@@ -125,6 +125,7 @@
    **** xref:policy-mule3-custom-policy-references.adoc[Configuration and Definition File Reference]
    **** xref:policy-mule3-pointcut-reference.adoc[Pointcut Reference]
    **** xref:policy-mule3-resource-level-custom-policy.adoc[Enable a Resource Level Support for a Custom Policy]
+   **** xref:upgrade-custom-policy-mule3.adoc[Change Versions of a Custom Policy]
```
Could we rename the url as well? Maybe `change-policy-mule3`

fermujica committed:
Change file name from 'upgrade' to 'change'

fermujica committed:
Add information for this change

sbehrends reviewed (approved):
LGTM 🚀
